src/DataSource/Interpreter/JsonFileInterpreter.php

```diff
@@ -52,7 +63,7 @@ protected function doInterpretFileAndCallProcessRow(string $path): void
 
     public function setSettings(array $settings): void
     {
-        //nothing to do
+        $this->path = $settings['path'];
```

Comment:
do we want to do any validation here that the path passed in is a valid JmesPath?